frontends/web/src/routes/accounts/all-accounts.tsx

```diff
@@ -48,13 +48,14 @@ const AccountItem = ({ account, hideAmounts }: { account: accountApi.IAccount, h
           return;
         }
         if (!balance.success) {
+          setBalance(undefined);
           return;
         }
         const balanceData = balance.balance;
         if (balanceData.hasAvailable) {
-          setBalance(balanceData.available.amount);
+          setBalance(balanceData.available);
         } else {
-          setBalance('0');
```

Comment:
@thisconnect @shonsirsha please note to never show a `0` if a balance is not loaded yet or unavailable. It is bad practice and can induce heart attacks :sweat_smile:

Reply:
Fixed up, PTAL again @thisconnect